### PR TITLE
chore(docs): updating docs for 3.0 launch

### PR DESCRIPTION
**Pull request by dbanksdesign**
*Issue #, if available:*

*Description of changes:* Updating some of the docs to reflect the 3.0 launch

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/version_3.md [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+* [Typescript support](#typescript-support)
+* [More built-ins](#more-built-ins)
+* [Bug fixes](#bug-fixes)
+* [Other features](#other-features)
+* [Better testing](#better-testing)
+* [Dropping support for older versions of node](#dropping-support-for-older-versions-of-node)
+
+### Style Properties → Design Tokens
+
+One last change is Style Dictionary is moving to the term "design tokens", both in documentation and in code. This has become the industry standard term for a while and it is time we respect that. Until now, Style Dictionary had called these "style properties" or just "properties", with some parts of the documentation also mentioning "design tokens". We want to be consistent with the direction of the community as well as in our documentation and code. We use the terms `properties` and `allProperties` in different APIs in Style Dictionary. To be consistent in documentation as well as code, we will be moving to using `tokens` and `allTokens`.
```
Comment (shaunbent) [inline docs/version_3.md:34]:
```suggestion
Style Dictionary is moving to the term "design tokens", both in documentation and in code. This has become the industry standard term for a while and it is time we respect that. Until now, Style Dictionary had called these "style properties" or just "properties", with some parts of the documentation also mentioning "design tokens". We want to be consistent with the direction of the community as well as in our documentation and code. We use the terms `properties` and `allProperties` in different APIs in Style Dictionary. To be consistent in documentation as well as code, we will be moving to using `tokens` and `allTokens`.
```

Feels like off to start the first point in the list with "One last change is..." maybe?

Comment (dbanksdesign) [reply docs/version_3.md:34]: 🤦 ah in my hastiness...

Comment (shaunbent) [reply docs/version_3.md]: My hastiness brought me here to see if V3 was out. 🤪

File: docs/version_3.md [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
+If you are starting a new project, you can install Style Dictionary and it will give you the latest version:
 
-`npm install --save-dev style-dictionary@next`
+`npm install --save-dev style-dictionary`
+
+If you have an existing project, you can upgrade to 3.0 by updating the version in your `package.json` file to `"style-dictionary": "^3.0.0"` and then run `npm install` or you can use the `latest` tag to update both your `package.json` and `package-lock.json` files:
+
+`npm install --save-dev style-dictionary@latest`
 
 If you find any bugs or issues, please file tickets so we can get things fixed before we release it to the wider community. Thanks!
```
Comment (chazzmoney) [inline docs/version_3.md:13]: > If you find any bugs or issues, please file tickets so we can get things fixed before we release it to the wider community. Thanks!

If you find any bugs or issues, please file tickets so we can get things fixed. We have tested and reviewed 3.0 extensively, but there may be things we missed. Thanks!

Comment (dbanksdesign) [reply docs/version_3.md]: Done and done!

File: docs/version_3.md [hunk 3]
```diff
@@ -1,12 +1,40 @@
...
 
-`npm install --save-dev style-dictionary@next`
+`npm install --save-dev style-dictionary`
 
-If you find any bugs or issues, please file tickets so we can get things fixed before we release it to the wider community. Thanks!
+If you have an existing project, you can upgrade to 3.0 by updating the version in your `package.json` file to `"style-dictionary": "^3.0.0"` and then run `npm install` or you can use the `latest` tag to update both your `package.json` and `package-lock.json` files:
+
+`npm install --save-dev style-dictionary@latest`
+
+If you find any bugs or issues, please file tickets so we can get things fixed. We have tested and reviewed 3.0 extensively, but there may be things we missed. Thanks!
```
Comment (chazzmoney) [inline docs/version_3.md:13]: can we add a link to file a ticket?

https://github.com/amzn/style-dictionary/issues/new

Comment (chazzmoney) [reply docs/version_3.md:13]: file tickets -> file a ticket

Comment (dbanksdesign) [reply docs/version_3.md]: done and done!

**Comment by chazzmoney** [comment]
LGTM! :shipit:

**Pull request merged**